runconfig/errors.go

```diff
@@ -38,3 +40,7 @@ var (
 	// ErrConflictUTSHostname conflict between the hostname and the UTS mode
 	ErrConflictUTSHostname = fmt.Errorf("Conflicting options: hostname and the UTS mode")
```

Comment:
These are all capitalized.  They should be lower-cased but not going to change them in this PR.